test/index.html

```diff
@@ -0,0 +1,247 @@
+      if ( !_impressSupported() ) {
+        assert.ok( notSupportedClass,
+                   "body.impress-not-supported class still there." );
+        assert.ok( !yesSupportedClass,
+                   "body.impress-supported class was NOT added." );
+      } else {
+        assert.ok( !notSupportedClass,
+                   "body.impress-not-supported class was removed." );
+        assert.ok( yesSupportedClass,
+                   "body.impress-supported class was added." );
+                   
+        // To be pedantic, we run the rest of these tests inside the else 
+        // brackets as well, meaning that impress.js is tested to be supported.
+        // However, other Qunit.test() blocks than this will fail miserably if it
+        // weren't supported.
+        assert.ok( !document.body.classList.contains("impress-disabled"),
+                   "body.impress-disabled is removed." );
+        assert.ok( document.body.classList.contains("impress-enabled"),
+                   "body.impress-enabled is added." );
+        
+        var canvas = document.querySelector( "div#impress > div" );
+        assert.ok( !canvas.classList.contains("step") && canvas.id === "",
+                   "Additional 'canvas' div inserted between div#impress root and steps." );
+        assert.equal( canvas.style.transform,
+                      "rotateZ(0deg) rotateY(0deg) rotateX(0deg) translate3d(1000px, 0px, 0px)",
+                      "canvas.style.transform initialized correctly" );
+        assert.equal( canvas.style.transformOrigin,
+                      "left top 0px",
+                      "canvas.style.transformOrigin initialized correctly" );
+        assert.equal( canvas.style.transformStyle,
+                      "preserve-3d",
+                      "canvas.style.transformStyle initialized correctly" );
+        assert.equal( canvas.style.transitionDelay,
+                      "0ms",
+                      "canvas.style.transitionDelay initialized correctly" );
+        // impress.js default values tries to set this to 1000ms, I'm completely confused about why that's not actually set in the browser?
+        assert.equal( canvas.style.transitionDuration,
+                      "0ms",
+                      "canvas.style.transitionDuration initialized correctly" );
+        assert.equal( canvas.style.transitionProperty,
+                      "all",
+                      "canvas.style.transitionProperty initialized correctly" );
+        assert.equal( canvas.style.transitionTimingFunction,
+                      "ease-in-out",
+                      "canvas.style.transitionTimingFunction initialized correctly" );
+                      
+        assert.equal( document.documentElement.style.height,
+                      "100%",
+                      "documentElement.style.height is 100%" );
+        
+        // Steps initialization
+        var step1 = document.querySelector( "div#step-1" );
```

Comment:
True. Checking impress.js for guidance, it seems itself to be very inconsistent.

The logic in my code above is that short words go without spaces, like ´contains("impress-enabled")´ and longer (which is most) arguments are surrounded by space inside parenthesis.

My proposal is to leave this discussion for now, and later do an overhaul of both impress.js and this code where style is settled once and for all. (Preferably in the same pull request with adding a style checker.) This is not to say I'm opposed to comments about style in general.